docs/examples/101/basic-batch-account/main.bicep

```diff
@@ -0,0 +1,20 @@
+param batchAccountName string
+param allocationMode string {
+  default: 'BatchService'
+  allowed: [
+    'BatchService'
+    'Usersubscription'

```

Comment:
You are getting a warning because the casing is not correct.  Change `'Usersubscription'` => `'UserSubscription'`.